src/Microsoft.ML.Core/Data/RoleMappedSchema.cs

```diff
@@ -263,38 +292,27 @@ private static void Add(Dictionary<string, List<ColumnInfo>> map, ColumnRole rol
         /// Returns whether there are any columns with the given column role.
         /// </summary>
         public bool Has(ColumnRole role)
-        {
-            return role.Value != null && _map.ContainsKey(role.Value);
-        }
+            => _map.ContainsKey(role.Value);
 
         /// <summary>
         /// Returns whether there is exactly one column of the given role.
         /// </summary>
         public bool HasUnique(ColumnRole role)
-        {
-            IReadOnlyList<ColumnInfo> cols;
-            return role.Value != null && _map.TryGetValue(role.Value, out cols) && cols.Count == 1;
-        }
+            => _map.TryGetValue(role.Value, out var cols) && cols.Count == 1;
 
         /// <summary>
         /// Returns whether there are two or more columns of the given role.
         /// </summary>
         public bool HasMultiple(ColumnRole role)
-        {
-            IReadOnlyList<ColumnInfo> cols;
-            return role.Value != null && _map.TryGetValue(role.Value, out cols) && cols.Count > 1;
-        }
+            => _map.TryGetValue(role.Value, out var cols) && cols.Count > 1;
 
         /// <summary>
         /// If there are columns of the given role, this returns the infos as a readonly list. Otherwise,
         /// it returns null.
         /// </summary>
         public IReadOnlyList<ColumnInfo> GetColumns(ColumnRole role)
         {
-            IReadOnlyList<ColumnInfo> list;
-            if (role.Value != null && _map.TryGetValue(role.Value, out list))
-                return list;
-            return null;
+            return _map.TryGetValue(role.Value, out var list) ? list : null;
```

Comment:
>return [](start = 12, length = 6)

it look strange to see all methods above get converted from return to  => and this one still use return.